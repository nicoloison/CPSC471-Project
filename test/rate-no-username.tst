curl localhost/recipes/rate.php
