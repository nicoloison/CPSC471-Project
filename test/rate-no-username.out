{"result":"failure","error":"missing parameter username"}